crates/utils/src/attr/types.rs

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
Also, macros are visible at the crate level is usually good to put them in a centralized `macros.rs` file merely use them here.